### PR TITLE
fdkaac: init at 1.0.0

### PR DESCRIPTION
**Pull request by lunik1**
###### Motivation for this change
Adds cli for fdk aac encoding. Needed as a dependency in upcoming patch adding [m4b-tool](https://github.com/sandreas/m4b-tool) (edit: #87957).

Currently has a very similar name to [fdk_aac](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/fdk-aac/default.nix), aka libfdk-aac. Open to suggestions on alternative names for this package if needed - fdkaac-cli maybe?

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by lunik1** [comment]
D'oh, forgot a capital at the beginning of the description!

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -4603,6 +4603,16 @@
     githubId = 26020062;
     name = "lumi";
   };
+  lunik1 = {
```
Comment (etu) [inline maintainers/maintainer-list.nix:4606]: The creation of this entry should be a separate commit

Comment (lunik1) [reply maintainers/maintainer-list.nix:4606]: Roger Roger

**Comment by etu** [comment]
@ofborg build fdkaac

**Review by etu — APPROVED** [review]
It seems to build and run fine for me, but as @lunik1 mentioned the name could be misleading.

I think the code as is is fine, but might be worth renaming the attribute to append `-cli` at the end of it to make the difference more clear.

**Comment by lunik1** [comment]
Possible alternative: homebrew calls this [`fdk-aac-encoder`](https://formulae.brew.sh/formula/fdk-aac-encoder).

**Comment by lunik1** [comment]
I've gone with `fdk-aac-encoder` since I figure it's better to share a name with one other package managment systems than zero other package management systems.
